Advanced/Sync Addressables/Assets/SyncAddressables/SyncBundleProvider.cs

```diff
@@ -23,10 +25,37 @@ internal void Start(ProvideHandle provideHandle)
                 Debug.LogError("the bundle failed " + provideHandle.Location.InternalId);
             provideHandle.Complete(this, m_AssetBundle != null, null);
         }
+
+        internal void Unload()
+        {
+            if (m_AssetBundle != null)
+            {
+                m_AssetBundle.Unload(true);
+                m_AssetBundle = null;
+            }
+        }
     }
     
     public override void Provide(ProvideHandle providerInterface)
     {
         new SyncAssetBundleResource().Start(providerInterface);
     }
+
+    public override void Release(IResourceLocation location, object asset)
+    {
+        if (location == null)
```

Comment:
This check will never trigger and can be safely removed. See ResourceManager.cs:374 - location's fields are accessed without null-check.